packages/ecma402-abstract/NumberFormat/PartitionNumberRangePattern.ts

```diff
@@ -57,7 +57,7 @@ export function PartitionNumberRangePattern(
   const symbols =
     internalSlots.dataLocaleData.numbers.symbols[internalSlots.numberingSystem]
 
-  const rangeSeparator = symbols.timeSeparator
+  const rangeSeparator = symbols.minusSign
```

Comment:
this seems wrong. It should come from https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-numbers-full/main/af/numbers.json#L142 so likely you'd have to change the data schema to handle this

Reply:
just looked at `af` and range separator is `&#8211;` while minus sign is a regular `-` so this is wrong

Reply:
I think that I can't simply change the data schema to handle this.
Currently, symbols are fetched from a structure [like this](https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-numbers-full/main/af/numbers.json#L16); as you can see, here the `rangeSeparator` symbol is not defined.

So I've tried to hardcode that value in the [extract-numbers script](https://github.com/formatjs/formatjs/blob/main/packages/intl-numberformat/scripts/extract-numbers.ts#L64), but then lots of tests were failing due to differences between JSON test data and the new format, even in the `test262` submodule.

At the end, as this symbol is used just once in this abstract function, I've hardcoded it there.

Let me know if there is a better way to proceed.

Reply:
the problem is that it's not a hardcoded symbol as I mentioned in `af` it's not `minusSign`. I think you can assumed the `rangeSeparator` comes from `range` pattern and is between `{0}` & `{1}` and write a corresponding script like `extract-numbers` and make a new file. Then you can import & use it in `core`

Reply:
The `minusSymbol` has been removed, this comment reference to an old version of the file.
Now, the `const rangeOperator` has been replaced with
```
result.push({type: 'literal', value: '–', source: 'shared'})
```

Searching inside the pattern range seems a little bit flaky, isn't it?

Reply:
ping

Reply:
this still doesn't solve my issue, you cannot hardcode `-`

Reply:
This has been addressed.